src/Image.php

```diff
@@ -25,6 +25,12 @@ class Image extends File
      */
     private static $plural_name = "Images";
 
+    /**
+     * @config
+     * @var bool
+     */
+    private static $lazy_load = true;
```

Comment:
Maybe it should say something like `$enable_lazy_loading`, as `$lazy_load` is a bit ambiguous.

Reply:
@emteknetnz Can you walk us through why you chose to put this config option here? What other alternative did you consider?

Reply:
Updated

Reply:
@maxime-rainville Both ImageShortcodeProvider.php and DBFile.php live in the assets module, so it needed to be a file within this module

Lazy loading only applies to Images, and not generic files.  Image.php lives in assets module and is the base class of, images :-)  So it seems pretty logical.   Other alternatives would be files in the assets src folder, though none seemed better than Image

Reply:
My instinct would have been to put it on a class where it's actually use like `DBFile`.

Reply:
ImageShortCodeProvider.php also uses it, so I'm not sure DBFile makes the most sense

It would look kind of weird putting this on ImageShortCodeProvider and then from DBFile calling `if (ImageShortCodeProvider::lazyLoadingEnabled()) {`

`Image::lazyLoadingEnabled()` reads the cleanest IMO